freyja_adapters/digital_twin/ibeji_adapter/src/config.rs

```diff
@@ -2,7 +2,32 @@
 // Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
+#![allow(unused_imports)]
+
 use serde::{Deserialize, Serialize};
+use std::env;
+
+/// If feature 'containerize' is set, will modify a localhost uri to point to container's localhost
+/// DNS alias. Otherwise, returns the uri as a String.
+///
+/// # Arguments
+/// * `uri` - The uri to potentially modify.
+pub fn get_uri(uri: &str) -> Result<String, Box<dyn std::error::Error + Send + Sync>> {
+    #[cfg(feature = "containerize")]
+    let uri = {
+        // Container env variable names.
+        let host_gateway_env_var: &str = "HOST_GATEWAY";
```

Comment:
Should these environment variable names be constants?

Reply:
I had changed this to reflect the change made in the Ibeji repo since these variables are only used in this function when the feature is enabled and nowhere else